src/hotspot/share/utilities/growableArray.hpp

```diff
@@ -254,10 +254,18 @@ class GrowableArrayView : public GrowableArrayBase {
 
   // Remove all elements up to the index (exclusive). The order is preserved.
   void remove_till(int idx) {
-    for (int i = 0, j = idx; j < length(); i++, j++) {
+    erase(0, idx);
+  }
+
+  // Remove all elements in the range [start - end). The order is preserved.
+  void erase(int start, int end) {
```

Comment:
I'd probably name this new method `remove_range` instead of using something completely unrelated.